src/nuget.config

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I like to use `src/Directory.build.props` instead of this file. It is enough to add there only the following:
```xml
<PropertyGroup>
    <RestoreSources>
        $(RestoreSources);
        https://www.myget.org/F/avalonia-ci/api/v2;
    </RestoreSources>
</PropertyGroup>
```

_As far as I remember, this only works for the sdk style_

Reply:
Thanks! This definitely looks better than a `nuget.config`. Going to implement this soon.